eng/Directory.Build.Mgmt.targets

```diff
@@ -1,4 +1,17 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(NetSdkBuildTargetsDir)\core\_Directory.Build.targets" Condition=" Exists('$(NetSdkBuildTargetsDir)\core\_Directory.Build.targets') "/>
   <Import Project="$(LibraryToolsFolder)\bootstrapTools\bootstrap.targets" />
+  <PropertyGroup>
```

Comment:
@shahabhijeet this change impacts that management libraries and was needed in order for the test project to reference and run against both the data-plane and management library. I briefly chatted with you about switching to public signing and I still suggest you do but I'd understand if you would like to hold on this change and if so we will need to disable these tests for now.